webapp/manifest.json

```diff
@@ -1,5 +1,5 @@
 {
-	"_version": "1.12.0",
+	"_version": "1.56.0",
```

Comment:
Are we looking for the latest schema version here?
Then it should be `1.57.0`. Please, check the spec here: https://raw.githubusercontent.com/SAP/ui5-manifest/master/schema.json